scripts/metrics/common/logging.py

```diff
@@ -0,0 +1,65 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Standardized logging util file for metrics scripts
+"""
+
+import enum
+import logging
+import sys
+
+
+class Level(enum.IntEnum):
+    CRITICAL = 50
```

Comment:
Why reinvent log levels when logging has defined log levels?

Reply:
This is using the same values as the defined log levels, but just a nicer enum to reference them. Was unsure if accessing the raw values would be preferable.

Reply:
I'm already working on a script which I want to use the same logging format. I can remove the enums if you want though.